modules/ossm-kiali-features.adoc

```diff
@@ -0,0 +1,23 @@
+////
+This CONCEPT module included in the following assemblies:
+-ossm-kiali.adoc
+////
+
+[id="ossm-kiali-features_{context}"]
+= Kiali features
+//In the title include nouns or noun phrases that are used in the body text.
+//Do not start the title of concept modules with a verb..
+
+The Kiali observability console is integrated with Red Hat Service Mesh and provides the following capabilities:
+
+* *Health* – Quickly identify issues with applications, services, or workloads.
+
+* *Topology* – Visualize how your applications, services, or workloads communicate via the Kiali graph.
+
+* *Metrics* – Predefined metrics dashboards let you chart service mesh and application performance for Go, Node.js. Quarkus, Spring Boot, Thorntail and Vert.x.  You can also create your own custom dashboards.
+
+* *Tracing* – Integration with Jaeger lets you follow the path of a request through various microservices that make up an application.
+
+* *Validations* – Perform advanced validations on the most common Istio Objects (Destination Rules, Service Entries, Virtual Services, and so on).
+
+* *Configuration* – Optional ability to create, update and delete Istio routing configuration via wizards in the Kiali Console.
```

Comment:
It also allows to update and delete Istio routing configuration directly in editor page outside of wizard.
"... via wizards or directly in editor page in the Kiali ...."

Reply:
verified